ldk/javascript/src/whisper/mapper.ts

```diff
@@ -120,6 +120,19 @@ export function mapToInternalChildComponent(
 export function mapToInternalComponent(component: Component): OliveHelps.Components {
   switch (component.type) {
     case WhisperComponentType.CollapseBox:
+      // eslint-disable-next-line
```

Comment:
Can you use the specific eslint rule disable for posterity :)

Reply:
Updated